src/Renderer/LayeredMaterial.js

```diff
@@ -76,7 +90,7 @@ var moveElementsArraySafe = function moveElementsArraySafe(array,index, howMany,
 };
 /* eslint-enable */
 
-const LayeredMaterial = function LayeredMaterial(id) {
+const LayeredMaterial = function LayeredMaterial(material, coord) {
```

Comment:
I would like to do. We need to talk about this.

-  `mapping` will not bear directly into uniforms